optimism.tokenlist.json

```diff
@@ -16,18 +16,13 @@
       "decimals": 18,
       "logoURI": "https://ethereum-optimism.github.io/logos/SNX.svg",
       "extensions": {
-        "optimismBridgeAddress": "0xCd9D4988C0AE61887B075bA77f08cbFAd2b65068"
-      }
-    },
-    {
-      "chainId": 10,
-      "address": "0x8700daec35af8ff88c16bdf0418774cb3d7599b4",
-      "name": "Synthetix",
-      "symbol": "SNX",
-      "decimals": 18,
-      "logoURI": "https://ethereum-optimism.github.io/logos/SNX.svg",
-      "extensions": {
-        "optimismBridgeAddress": "0x3f87Ff1de58128eF8FCb4c807eFD776E1aC72E51"
+        "bridgeInfo": {
```

Comment:
i think dapps may expect to see `chainId: 10` at root level as opposed to only in `extensions`.

Reply:
Sure, this is possible for L2 native tokens like these https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/2138386277e4156d159615d1840882cecc398437/optimism.tokenlist.json#L300
If a token is L1 native though it makes more sense its root node in the schema here to be for `chainId: 1`. It's more optimal normalisation of the data as opposed to the flat schema option.